src/features/commentsSlice.ts

```diff
@@ -0,0 +1,72 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const initComments = createAsyncThunk(
+  'comments/fetch',
+  async (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const initAddComment = createAsyncThunk(
+  'comments/addComment',
+  async (comment: Omit<Comment, 'id'>) => {
+    return createComment(comment);
+  },
+);
+
+export const initDeleteComment = createAsyncThunk(
+  'comments/removeComment',
+  (commentId: number) => {
+    deleteComment(commentId);
```

Comment:
The `initDeleteComment` thunk lacks error handling. Consider wrapping the `deleteComment` call in a try-catch block to handle potential errors and return a rejected action if an error occurs.